web-main/src/app/state/services/con-api.service.ts

```diff
@@ -44,6 +44,29 @@ const IN_MEMORY_MSG_LIST: Partial<Record<models.Conversation.Id, readonly models
 
 @ngCore.Injectable()
 export class ConApiService {
+  public readonly msgReceived$ = new rxjs.Subject<models.Conversation.Message.InContext>()
+
+  public sendConMessage(payloadMessage: models.Conversation.Message.InContext.Input) {
```

Comment:
Consider what this method does: 
- it calls back-end server, that we've send a mesage. 
- back-end server then returns a new message, through some kind of `hub` method, and that `hub` method then calls "this.msgRecived$.next(newMessage)" to simulate that, we will do that in our "sendConMessage" method, so the implementation of the `sendConMessage` should be following: 

```typescript
 public sendConMessage(payloadMessage: models.Conversation.Message.InContext.Input) {
   const newMessage = {...payloadMessage, id: new new Date().getTime().toString()}
   
   // optionally you can store message on this "IN_MEMORY_MSG_LIST" but it is not needed, 
  // as we no longer gonna pull anything from it, after intial load, so it is only importat that after that time, we just updat ethe state
   
   //TODO:  simulating that we recived message from the back-end, this will be removed in the future, once we have  a back-end
   this.msgRecived.next(newMessage)
 }
 ```